rdbms/src/main/java/io/spine/server/storage/jdbc/JdbcStorageFactory.java

```diff
@@ -211,13 +225,36 @@ public Builder setDataSource(DataSourceConfig dataSource) {
             return this;
         }
 
+        /**
+         * Sets {@link TypeMapping}, which defines {@link Type} names for the used database.
+         *
+         * <p>Use the {@linkplain BaseMapping#baseBuilder() base builder} to build a custom mapping.
+         *
+         * <p>If the mapping was not specified, it is
+         * {@linkplain StandardMapping#select(DataSourceWrapper) selected} basing on
+         * the {@linkplain java.sql.DatabaseMetaData#getDatabaseProductName() database product name}
+         * and the {@linkplain java.sql.DatabaseMetaData#getDatabaseMajorVersion() major version}.
+         *
+         * <p>If there is no mapping for the database,
+         * MySQL-specific mapping for the 5th version is used.
```

Comment:
Instead of this, please give a link to the default value.

Also, there isn't a `5th version`. There are `5.0`, `5.1` etc.

Reply:
Done.